rukundoeric/calculator
# Milestone 2: Components

## What does this do? 
*  Calculator
## Description of Task to be completed? 
*  In this milestone I implemented all components
## How should this be manually tested? 
NOTE: Make sure you have Node.js and npm or yarn installed on your PC.
- Clone this repo: `git clone https://github.com/rukundoeric/calculator.git`
- Navigate into the project directory: `cd calculator`
- Install all packages: `yarn install or npm install`
- Start the app: `yarn start or npm start`
